contracts/contracts/poolBooster/PoolBoosterSwapxIchi.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { IBribe } from "../interfaces/poolBooster/ISwapXAlgebraBribe.sol";
+import { IPoolBooster } from "../interfaces/poolBooster/IPoolBooster.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { StableMath } from "../utils/StableMath.sol";
+
+/**
+ * @title Pool booster for SwapX concentrated liquidity Ichi vault enabled pools
+ * @author Origin Protocol Inc
+ */
+contract PoolBoosterSwapxIchi is IPoolBooster {
+    using StableMath for uint256;
+
+    // @notice address of the Bribes.sol(Bribe) contract for the OS token side
+    IBribe public immutable bribeContractOS;
+    // @notice address of the  Bribes.sol(Bribe) contract for the other token in the pool
+    IBribe public immutable bribeContractOther;
+    // @notice address of the OS token
+    IERC20 public immutable osToken;
+    // @notice 1e18 denominated split between OS and Other bribe. E.g. 0.4e17 means 40% to OS
+    //         bribe contract and 60% to other bribe contract
+    uint256 public immutable split;
+
+    // @notice if balance under this amount the bribe action is skipped
+    uint256 public constant MIN_BRIBE_AMOUNT = 1e10;
+
+    constructor(
+        address _bribeContractOS,
+        address _bribeContractOther,
+        address _osToken,
```

Comment:
The osToken should be oeth, ousd... tokens with 18 decimals. Maybe just make sure it's not a token with less decimals that is added to make sure `MIN_BRIBE_AMOUNT` is still valid

Reply:
Our tokens always have 18 decimals, and if we were to break that rule, we'd have problems to address all through the codebase. Still good comment, thanks.